vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java

```diff
@@ -67,11 +67,19 @@ public KeyboardActions(boolean realDevice, TouchActions touchActions,
      * </ol>
      * @param element element to type text, must not be {@code null}
      * @param text text to type into the element, must not be {@code null}
+     * @param hideKeyboard flag showing whether to hide the keyboard after typing
      */
-    public void typeText(WebElement element, String text)
+    public void typeText(WebElement element, String text, boolean hideKeyboard)
```

Comment:
since we are changing the API, I propose to avoid unnecessary if else's to have two separate methods, first one will hide the keyboard after type, the second one will leave it open. 

cc @valfirst  @uarlouski

Reply:
![image](https://user-images.githubusercontent.com/17816729/134188527-1f7fa5d4-9a59-4cbf-abdc-8cab959172ae.png)

Reply:
^^ as an example

Reply:
Done.